plugin/http/httptrace/httptrace.go

```diff
@@ -0,0 +1,120 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package httptrace contains OpenCensus tracing integrations with net/http.
+package httptrace
+
+import (
+	"net/http"
+	"strings"
+
+	"go.opencensus.io/trace"
+	"go.opencensus.io/trace/propagation"
+)
+
+const httpHeader = `X-Cloud-Trace-Context`
+
+type transport struct {
+	base        http.RoundTripper
+	propogators []propagation.HTTPPropagator
+}
+
+// RoundTrip creates a trace.Span and inserts it into the outgoing request's headers.
+// The created span can follow a parent span, if a parent is presented in
+// the request's context.
+func (t *transport) RoundTrip(req *http.Request) (*http.Response, error) {
+	name := "Sent" + strings.Replace(req.URL.String(), req.URL.Scheme, ".", -1)
```

Comment:
The name here strikes me as odd because "sent" seems like the name of an event rather than a description of the span itself. Is this standardized somewhere? I would be less surprised by a name starting with "Client" or "HTTP"

Reply:
I think this is the kind of thing that should be documented across languages in census-instrumentation/opencensus-specs#13